modules/sharedaddy/sharing.js

```diff
@@ -42,8 +42,12 @@ if ( sharing_js_options && sharing_js_options.counts ) {
 						continue;
 					}
 
-					while ( ( service_url = urls[ service ].pop() ) ) {
-						jQuery.getScript( service_url );
+					while ( ( service_request = requests[ service ].pop() ) ) {
```

Comment:
These changes can be omitted, since Twitter requests were the only non-string requests.